mapping/sustainability.yaml

```
@@ -1186,7 +1214,7 @@
     title: Reporting period
     module: Climate finance
     indicator: KPIs
-    disclosure format: It could be quarterly, annually, biannually, etc. \[free text\].
+    disclosure format: Clarify the project reporting period, it could be quarterly, annually, biannually.
```

```suggestion
    disclosure format: Clarify the project reporting period. It could be quarterly, annually, biannually.
```
This needs to be changed in the Data elements sheet as well.